src/lib/parameters/parameters.cpp

```diff
@@ -965,11 +964,14 @@ param_save_default()
 		return res;
 	}
 
+	px4_shutdown_lock();
```

Comment:
Nice!

Reply:
Other calls always check the return value, is that unnecessary?

Reply:
Yes, I suppose they should be paired.

Reply:
Done.